src/sentry/static/sentry/app/components/tooltip.tsx

```diff
@@ -290,7 +290,7 @@ const Container = styled('span')<{
 `;
 
 const TooltipContent = styled('div')<{hide: boolean} & Pick<Props, 'popperStyle'>>`
-  color: #fff;
+  color: ${p => p.theme.white};
```

Comment:
⭐️

Reply:
@robinrendle Looking ahead to dark mode, should this be `theme.background`? Or do we want to add an alias like `inverseTextColor`?

Reply:
Ah. Mmmm...I think a more semantic name like`inverseTextColor` works. Makes it more obvious to me and I think there’s a bunch of places where we’d want to use that.